### PR TITLE
feat: Add brand moment animations

### PR DESCRIPTION
**Pull request by ActuallyACat**
# What this PR does: 
* Adds new Brand Moments components: https://dev.cultureamp.design/ally/webm-player/storybook/iframe.html?id=illustration-scene-react--brand-moments&viewMode=story
* Adds new Empty states components: https://dev.cultureamp.design/ally/webm-player/storybook/iframe.html?id=illustration-scene-react--animated-scene-illustrations&viewMode=story
* All animations are optional 
* If there is an error with an asset it will fallback to a static image 

Progressive enhancement / future improvements: 
* Capture intro component is missing the initial animation
* Abstract repeated logic into a HOC
* Should `BrandMomentCaptureIntro` be a snowflake since it's the only animation with both ambient and initial animations? 
* Generate mp4 for our safari-browser friends.

# Motivation and Context

# Screenshots (if appropriate)

# Checklist

- [ ] If this contains visual changes, has it been reviewed by a designer?
- [ ] If this introduces either a new component or a breaking change to an existing component, has it been reviewed by an advocate? (Ask in #prod_design_systems)
- [ ] I have considered likely risks of these changes and got someone else to QA as appropriate
- [ ] I have or will communicate these changes to the front end practice

# Additional Considerations
- Does there need to be right-to-left (RTL) options for localization and internationalization?
- Has the test suite been updated?
- Have you done cross-browser testing for our [supported browsers](https://academy.cultureamp.com/hc/en-us/articles/204539569-Supported-browsers-for-Participants)?
- Have you updated any relevant documentation or left useful comments in the code?
- Have you reviewed Culture Amp's Web Accessibility guide [Current Compliance, Going Forward, Statement and Answers for Customers](https://cultureamp.atlassian.net/wiki/spaces/Prod/pages/428572998/Web+Accessibility)?
- If this contains substantial visual changes, especially far reaching ones, have you unlocked the Chromatic step in the pipeline?
- Have Storybook stories been updated?

**If you're new to Kaizen, please ask #prod_design_systems to set up an onboarding session to get you up to speed.** If you have an urgent PR to merge before that happens, it is safest to ask in #prod_design_systems to have a design system advocate review the PR to catch any issues.

### CODE REVIEW

File: draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      videoElement.setAttribute("muted", "")
+    }
+  }, [])
+
+  useEffect(() => {
+    if (!reducedMotionQuery.addEventListener || !window) return
+    const updateMotionPreferences = () => {
+      const { matches = false } = window.matchMedia(
+        "(prefers-reduced-motion: reduce)"
+      )
```
Comment (dougmacknz) [inline draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.tsx:79]: Is this done twice on purpose? (once on line 50 and again here)

File: draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+      width="100%"
+      data-testid="kz-video-player"
+      className={styles.wrapper}
+      loop={loop}
+      poster={assetUrl(fallback)}
+      autoPlay={prefersReducedMotion ? false : autoplay}
+      playsInline={true}
+    >
+      <source src={assetUrl(ambientAnimation)} type="video/webm" />
+      Sorry, your browser doesn't support embedded videos.
```
Comment (dougmacknz) [inline draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.tsx:124]: Not a big deal but I wonder if displaying nothing is a better fallback here, given the illustration is just for presentational purposes.

**Review by dougmacknz — APPROVED** [review]
So good! Left a couple comments but nothing blocking 🚢

File: draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.tsx (hunk 1 above)
Comment (ActuallyACat) [reply draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.tsx:79]: Yup - I do an initial query to get the users preferences then set up this listener in the (fairly uncommon) scenario of someone updating their motion preferences after this component has been rendered.

File: draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.tsx (hunk 2 above)
Comment (ActuallyACat) [reply draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.tsx:124]: Yeah - agreed. I'll fix this up.

**Review by dougmacknz — APPROVED** [review]

**Pull request merged**